nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -1165,13 +1168,22 @@ public void start(final boolean monitor) throws IOException {
 
         nifiPropsFilename = nifiPropsFilename.trim();
 
+        String xmx = null;
+        String xms = null;
+
         final List<String> javaAdditionalArgs = new ArrayList<>();
         for (final Map.Entry<String, String> entry : props.entrySet()) {
             final String key = entry.getKey();
             final String value = entry.getValue();
 
             if (key.startsWith("java.arg")) {
                 javaAdditionalArgs.add(value);
+                if(value.toLowerCase().startsWith("-xms")) {
```

Comment:
```suggestion
                if (value.toLowerCase().startsWith("-xms")) {
```

Reply:
Is there a reason for the `toLowerCase()` comparison? The case should be consistent.

Reply:
One could be configuring NiFi using -Xms and -Xmx, no? The default being:
````
java.arg.2=-Xms1g
java.arg.3=-Xmx1g
````

Reply:
OK I understand now... this is case sensitive so I can directly check against Xmx and Xms. Updating.

Reply:
Thanks!